dask/array/tests/test_fft.py

```diff
@@ -108,3 +113,41 @@ def test_fft_consistent_names():
     assert same_keys(fft(darr, 5), fft(darr, 5))
     assert same_keys(fft(darr2, 5, axis=0), fft(darr2, 5, axis=0))
     assert not same_keys(fft(darr, 5), fft(darr, 13))
+
+
+def test_wrap_numpy():
```

Comment:
Can we refactor this to use `pytest.mark.parametrize`?  This might make it easier to extend in the future.

```python
@pytest.mark.parametrize('module', [np.fft, scipy.fft])
@pytest.mark.parametrize('funcname', ['fft', 'ifft', 'rfft', ...])
def test_wrapper(module, funcname):
    try:
        func = getattr(module, funcname)
    except AttributeError:
        pytest.mark.skip()
    dfft = fft_wrapper(module, funcname)
    assert_eq(fft(nparr), dfft(darr))
```